scripts/run-checkout-tests.sh

```diff
@@ -40,6 +40,7 @@ assert_eq $(git ls-remote --heads origin | grep refs/heads/releases/v1 | cut -f1
 popd
 
 clean_up
+exit 0
```

Comment:
I think this might be a leftover?

Reply:
Yes! That was trying to get the error messages for #126 !

Removed now.